Source/Scene/ImplicitTileCoordinates.js

```diff
@@ -25,7 +25,8 @@ import ImplicitSubdivisionScheme from "./ImplicitSubdivisionScheme.js";
  *
  * @param {Object} options An object with the following properties:
  * @param {ImplicitSubdivisionScheme} options.subdivisionScheme Whether the coordinates are for a quadtree or octree
- * @param {Number} options.level The level of a tile relative to the tile with the extension.
+ * @param {Number} options.subtreeLevels The number of distinct levels within the coordinate's subtree
```

Comment:
Adding `subtreeLevels` made a lot of the new helper functions possible. This has been updated everywhere that uses `ImplicitTileCoordinates`